connectors/cassandra/schema.go

```diff
@@ -0,0 +1,232 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cassandra
+
+import (
+	"context"
+	"fmt"
+
+	"bytes"
+
+	"github.com/gocql/gocql"
+	"github.com/pkg/errors"
+	"github.com/uber-go/dosa"
+)
+
+// CreateScope creates a keyspace
+func (c *Connector) CreateScope(ctx context.Context, scope string) error {
+	// drop the old scope, ignoring errors
+	_ = c.DropScope(ctx, scope)
+
+	ksn := CleanupKeyspaceName(scope)
```

Comment:
In here, and in `DropScope`, we directly map the the scope to the keyspace name. But in `CheckSchema` we actually use the `KeySpace` mapper to determine the keyspace to use.

I'm guessing this has to do with some of the other problems related around keyspace, scope, and prefix mapping that we've talked about offline. Can we at least leave a `TODO` here to let people know that this inconsistent behavior is going to be fixed soon?

(meant to leave this comment in the previous review, but it too seemed to have gotten dropped some how).